Questie.lua

```diff
@@ -1,62 +1,31 @@
 -- When adding a new level here it MUST be assigned a number and name in `debugLevel.values` as well added to Questie:Debug below
-DEBUG_CRITICAL = "|cff00f2e6[CRITICAL]|r"
-DEBUG_ELEVATED = "|cffebf441[ELEVATED]|r"
-DEBUG_INFO = "|cff00bc32[INFO]|r"
-DEBUG_DEVELOP = "|cff7c83ff[DEVELOP]|r"
-DEBUG_SPAM = "|cffff8484[SPAM]|r"
+Questie.DEBUG_CRITICAL = "|cff00f2e6[CRITICAL]|r"
+Questie.DEBUG_ELEVATED = "|cffebf441[ELEVATED]|r"
+Questie.DEBUG_INFO = "|cff00bc32[INFO]|r"
+Questie.DEBUG_DEVELOP = "|cff7c83ff[DEVELOP]|r"
+Questie.DEBUG_SPAM = "|cffff8484[SPAM]|r"
 
+local band = bit.band
 
 -------------------------
 --Import modules.
 -------------------------
----@type QuestieSerializer
-local QuestieSerializer = QuestieLoader:ImportModule("QuestieSerializer");
 ---@type QuestieComms
 local QuestieComms = QuestieLoader:ImportModule("QuestieComms");
 ---@type QuestieOptions
 local QuestieOptions = QuestieLoader:ImportModule("QuestieOptions");
 ---@type QuestieOptionsDefaults
 local QuestieOptionsDefaults = QuestieLoader:ImportModule("QuestieOptionsDefaults")
----@type MinimapIcon
-local MinimapIcon = QuestieLoader:ImportModule("MinimapIcon");
----@type QuestieOptionsUtils
-local QuestieOptionsUtils = QuestieLoader:ImportModule("QuestieOptionsUtils");
----@type QuestieAuto
-local QuestieAuto = QuestieLoader:ImportModule("QuestieAuto");
 ---@type QuestieCoords
 local QuestieCoords = QuestieLoader:ImportModule("QuestieCoords");
 ---@type QuestieEventHandler
 local QuestieEventHandler = QuestieLoader:ImportModule("QuestieEventHandler");
----@type QuestieJourney
-local QuestieJourney = QuestieLoader:ImportModule("QuestieJourney");
----@type QuestieMap
-local QuestieMap = QuestieLoader:ImportModule("QuestieMap");
----@type QuestieNameplate
-local QuestieNameplate = QuestieLoader:ImportModule("QuestieNameplate");
----@type QuestieProfessions
-local QuestieProfessions = QuestieLoader:ImportModule("QuestieProfessions");
----@type QuestieQuest
-local QuestieQuest = QuestieLoader:ImportModule("QuestieQuest");
----@type QuestieReputation
-local QuestieReputation = QuestieLoader:ImportModule("QuestieReputation");
----@type QuestieSearch
-local QuestieSearch = QuestieLoader:ImportModule("QuestieSearch");
----@type QuestieSearchResults
-local QuestieSearchResults = QuestieLoader:ImportModule("QuestieSearchResults");
----@type QuestieStreamLib
-local QuestieStreamLib = QuestieLoader:ImportModule("QuestieStreamLib");
 ---@type QuestieTooltips
 local QuestieTooltips = QuestieLoader:ImportModule("QuestieTooltips");
----@type QuestieTracker
-local QuestieTracker = QuestieLoader:ImportModule("QuestieTracker");
 ---@type QuestieDBMIntegration
 local QuestieDBMIntegration = QuestieLoader:ImportModule("QuestieDBMIntegration");
----@type QuestieLib
-local QuestieLib = QuestieLoader:ImportModule("QuestieLib");
----@type QuestiePlayer
-local QuestiePlayer = QuestieLoader:ImportModule("QuestiePlayer");
```

Comment:
Uff, how the hell was this unnoticed for so long? 🙈 Good catch Laume! 👍🏻

Reply:
Because I removed need for those only a month ago <https://github.com/Questie/Questie/commit/d8b545bd7cbcb5887b2c5066d62fef1d485ecbb5> 🤫